specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+|                           | Machine Learning                  | Creation, training, and deployment of statistical algorithms that learn from and perform tasks based on data.                                                                                          |
+|                           | Natural Language Processing       | Generation of human language, handling tasks like translation, sentiment analysis, and text summarization.                                                                                             |
+|                           | Other (AI and Machine Learning)   | AI and Machine Learning services that do not fall into one of the defined subcategories.                                                                                                               |
+| Analytics                 | Analytics Platform                | Unified solution that combines technologies to meet enterprise needs across the entire analytics lifecycle.                                                                                            |
+|                           | Business Intelligence             | Semantic models, dashboards, reports, and data visualizations to track performance and identify trends.                                                                                                |
+|                           | Data Processing                   | Integration and transformation tasks to prepare data for analysis.                                                                                                                                     |
+|                           | Search                            | Discovery of information by indexing and retrieving data from various sources.                                                                                                                         |
+|                           | Streaming Analytics               | Real-time data stream processes to detect patterns, trends, and anomalies as they occur.                                                                                                               |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                                                                                                                   |
```

Comment:
Why isn't Bare Metal a sub-category of Compute?

Reply:
we focused on defining those subcategories that have the most commonality across all providers.  In the specific case of bare metal, it could be misleading to maintain such a subcategory, given that it would not be inclusive of bare metal activity across all providers.  For example, AWS bare metal instance types live under the EC2 service, which is categorized as `Virtual Machines`.

It is my belief that the definition of a _SKU_ taxonomy will handle for practitioner analyses such as these, and the _Service_ taxonomy cannot provide that kind of fine grain.

As it stands, there would be no appropriate subcategory for OCI Bare Metal Servers. Please keep in mind that there are many, many services across all the providers that will not have a subcategory home in the first version of this taxonomy, as a guiding principle here is to undercategorize rather than overcategorize and allow ourselves (and the community) to advocate for adding missing subcategories over time.

Reply:
Discussed this at length on the Tue Sep 3 TF1 call.  Group decided to subcategorize bare-metal under `Virtual Machines` and leave the name as-is.  Resolving.